pkg/session/session.go

```diff
@@ -117,7 +118,11 @@ func GetOrCreate(ctx context.Context, params *Params) (*Session, error) {
 	sessionMU.Lock()
 	defer sessionMU.Unlock()
 
-	sessionKey := params.server + params.userinfo.Username() + params.datacenter
+	userPassword, _ := params.userinfo.Password()
+	sessionName := params.server + params.datacenter + params.userinfo.Username() + userPassword
+	h := sha256.New()
+	h.Write([]byte(sessionName))
+	sessionKey := string(h.Sum(nil))
```

Comment:
```suggestion
	sessionKey := fmt.Sprintf("x", h.Sum(nil))
```

To actually get something human readable.

https://go.dev/play/p/ZIY3RMdXg-L

Reply:
Will update in next patch.